sys-apps/input-utils/input-utils-1.3.ebuild

```diff
@@ -1,21 +1,20 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 MY_PN="input"
 MY_P="${MY_PN}-${PV}"
 
 DESCRIPTION="Small collection of linux input layer utils"
 HOMEPAGE="https://www.kraxel.org/blog/linux/input/"
 SRC_URI="https://www.kraxel.org/releases/input/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ppc ~riscv x86"
 
-S="${WORKDIR}/${MY_P}"
-
 src_install() {
 	emake install bindir="${D}"/usr/bin mandir="${D}"/usr/share/man STRIP=""
```

Comment:
Don't we need here EPREFIX? Or does the build system does the rest?

Reply:
Hmm, I think you're right - but actually i'm not sure. The build sets the folders as following:
```
prefix  ?= /usr/local
bindir  =  $(DESTDIR)$(prefix)/bin
shrdir  =  $(DESTDIR)$(prefix)/share
mandir  =  $(shrdir)/man
```
I've added `EPREFIX="${EPREFIX}/usr"`, but i guess i also have to change `${D}` to `${ED}`?

Reply:
Yes, this would make sense.